sdk/core/core-rest-pipeline/src/nodeHttpClient.ts

```diff
@@ -231,6 +231,7 @@ class NodeHttpClient implements HttpClient {
       port: url.port,
       method: request.method,
       headers: request.headers.toJSON({ preserveCase: true }),
+      ...request.requestOverrides,
```

Comment:
Nit, you might want to do `...request.requestOverrides || {}` here if the requestOverrides is undefined.